app/src/main/java/com/internship/move/utils/InternalStorageManager.kt

```diff
@@ -3,28 +3,36 @@ package com.internship.move.utils
 import android.content.Context
 import android.content.SharedPreferences
 import com.internship.move.presentation.authentification.register.RegisterFragment
-import com.internship.move.presentation.map.MapFragment
+import com.internship.move.presentation.authentification.viewmodel.AuthenticationViewModel
+import com.internship.move.presentation.menu.MenuFragment
 
 class InternalStorageManager(
     context: Context
 ) {
     private val sharedPreferences: SharedPreferences =
         context.getSharedPreferences(KEY_APP_PREFERENCES, Context.MODE_PRIVATE)
 
-    fun getIsUserLoggedIn(): Boolean = sharedPreferences.getBoolean(MapFragment.KEY_IS_USER_LOGGED_IN, false)
-
+    fun getIsUserLoggedIn(): Boolean = sharedPreferences.getBoolean(MenuFragment.KEY_IS_USER_LOGGED_IN, false)
```

Comment:
I think these key constants should be in the `InternalStorageManager` companion object. Since it is the class that handles sharedPref, it should be the one who knows about them